Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -9,43 +9,44 @@ import Mathlib.Analysis.Calculus.Deriv.Inv
 import Mathlib.Analysis.SpecialFunctions.Pow.Deriv
 
 /-!
-# Divide-and-conquer recurrences and the Akra-Bazzi theorem
-
-A divide-and-conquer recurrence is a function `T : ℕ → ℝ` that satisfies a recurrence relation of
-the form `T(n) = ∑_{i=0}^{k-1} a_i T(r_i(n)) + g(n)` for large enough `n`, where `r_i(n)` is some
-function where `‖r_i(n) - b_i n‖ ∈ o(n / (log n)^2)` for every `i`, the `a_i`'s are some positive
-coefficients, and the `b_i`'s are reals `∈ (0,1)`. (Note that this can be improved to
-`O(n / (log n)^(1+ε))`, this is left as future work.) These recurrences arise mainly in the
+# Divide-and-conquer recurrences and the Akra–Bazzi theorem
+
+A divide-and-conquer recurrence is a function $T : ℕ → ℝ$ that satisfies a recurrence relation of
+the form
+$$T(n) = \sum_{i=0}^{k-1} a_i\, T(r_i(n)) + g(n),$$
+for sufficiently large $n$, where $r_i(n)$ is a function with
```

Comment:
I'm positive about half of the changes here, and negative about the others: when unicode can express the expression you want, I actually prefer this in the docs (as it renders fine enough in the generated docs, but renders *nicely* in VS Code, hovers, etc). Changing `n` or `T : ℕ → ℝ` is thus a regression for me.
The long formula which is clearly LaTeX makes sense as LaTeX.

I'd write `aᵢ` instead of $a_i$.

Please audit the other comment changes from the same angle, thanks!

Reply:
Rendering in the editor clearly matters, so I understand the preference for unicode over LaTeX.

I'm not sure I understand your implicit rules for when to use code highlighting (\`\`), and when not, though. I thought code highlighting was intended for when one is writing actual code, as in Lean? Why not just "aᵢ" (unicode, not LaTeX) instead of `aᵢ`?